### DISCUSSION TITLE
segmentation tools

### DISCUSSION THREAD

**Discussion by nikparastf** [kschan0214/sepia#18] [Q&A]
hello
i have one question about segmentation...
which tools or software do u prefer to segment susceptibility map?automatic ROIs detections on T1 ??
i know about FSL, but it is hard for me to work with it...
can u suggest me easier software?
and which tools do u prefer for analysis ROIs??
thanks

**Comment by kschan0214** [comment]
Hi, 

I think it depends on the tissue you want to study, the way to segment a tumour can be quite different compared to normal tissue segmentation. I mainly use FSL to segment GM/WM/subcortical GM on T1w image. I believe there exists many great tools that perform segmentation on T1/T1w images but I don't have enough expertise to give you an informed answer. Personally I also use thresholding/morphological operation and (or) sometimes ITK-snap for manual segmentation. 

(P.S. I converted this issue to the discussion section since it is not directly related to the usage of SEPIA)
